frontend/app/src/people/widgetViews/UserTicketsView.tsx

```diff
@@ -121,7 +120,7 @@ const UserTickets = () => {
         );
       })
     ) : (
-      <NoResults loading={loading} />
+      <NoResults />
```

Comment:
why was this loading prop removed?

Reply:
Because it was not being used, the loading functionality was done in the UserTicketsView component